tests/parse/test_parse_case.py

```diff
@@ -54,30 +54,39 @@ def test_parse_case_date(scout_config):
     assert isinstance(case_data["analysis_date"], datetime)
 
 
-def test_parse_case_owner(scout_config):
-    # GIVEN you load sample information from a scout config
-    # WHEN case is parsed
-    case_data = parse_case(scout_config)
-    # THEN the case should have a owner
-    assert case_data["owner"] == scout_config["owner"]
-
-
-def test_parse_case_limsid(scout_config):
-    """Test parsing a case with lims_id"""
-
+@pytest.mark.parametrize(
+    "param_name",
+    [
+        "analysis_date",
+        "delivery_report",
+        "gene_panels",
+        "lims_id",
+        "owner",
+        "peddy_ped",
+        "rank_model_version",
+        "rank_score_threshold",
+        "smn_tsv",
```

Comment:
There is a dedicated test to check the parsing of this single param at the end of this file: https://github.com/Clinical-Genomics/scout/blob/3adaaf7383b2dd8df2c2e0d6c89b85305deafd91/tests/parse/test_parse_case.py#L454

It can probably be removed now!